### PR TITLE
[FIX] New kudu server deployment

### PR DESCRIPTION
**Pull request by Stefan365**
//test: `@integrations-kudu`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: utilities/src/main/java/io/syndesis/qe/resource/impl/Kudu.java [hunk 1]
```diff
@@ -21,26 +22,76 @@
...
+
+                .editOrNewSpec()
+                .addToSelector(LABEL_NAME, APP_NAME)
+                .withReplicas(1)
+                .editOrNewTemplate()
+                .editOrNewMetadata()
+                .addToLabels(LABEL_NAME, APP_NAME)
+                .endMetadata()
+                .editOrNewSpec()
+                .addNewContainer().withName(APP_NAME).withImage("syndesisqe/kudu-2in1:latest")
```
Comment (avano) [inline utilities/src/main/java/io/syndesis/qe/resource/impl/Kudu.java]: where are the dockerfiles for these containers located?

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: utilities/src/main/java/io/syndesis/qe/resource/impl/Kudu.java [hunk 2]
```diff
@@ -21,26 +22,76 @@
...
+
+                .addAllToPorts(ports)
+                .addAllToEnv(templateParams)
+
+                .withNewSecurityContext()
+                .endSecurityContext()
```
Comment (avano) [inline utilities/src/main/java/io/syndesis/qe/resource/impl/Kudu.java]: these should not be needed

File: utilities/src/main/java/io/syndesis/qe/resource/impl/Kudu.java (hunk 1 above)
Comment (Stefan365) [reply utilities/src/main/java/io/syndesis/qe/resource/impl/Kudu.java]: here: git clone git@github.com:lgarciaaco/kudu-docker.git

Comment (Stefan365) [reply utilities/src/main/java/io/syndesis/qe/resource/impl/Kudu.java]: //1. clone kudu-docker project:
git clone git@github.com:lgarciaaco/kudu-docker.git

//2. in kudu-docker project, docker-entrypoint.sh, add to KUDU_TSERVER_OPTS= these 2 lines:
`   -memory_limit_hard_bytes=7294967296 \
   -block_cache_capacity_mb=1024 \`

//3. change ENTRYPOINT in kudu-docker, Dockerfile:
`ENTRYPOINT ["/docker-entrypoint.sh" , "single"]`

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: utilities/src/main/java/io/syndesis/qe/resource/impl/Kudu.java (hunk 1 above)
Comment (Stefan365) [reply utilities/src/main/java/io/syndesis/qe/resource/impl/Kudu.java]: kudu rest api 'how to do' is here:
git@github.com:mcada/kudu-rest-app.git

**Pull request merged**